content/References/ExtensionCalculator.md

```diff
@@ -19,6 +19,10 @@ Extended VDEVs recover lost headroom as existing data is read and rewritten to t
 This can occur naturally over the lifetime of the pool as you modify or delete data.
 To manually recover capacity, simply replicate and rewrite the data to the extended pool.
 
+Note: While this process can recover the actual lost capacity, reported capacity continues to rely on the old data-to-parity ratio.
+An expanded vdev can continue to report a lower than expected capacity, even after rewriting old data to the new parity ratio.
+This accounting inconsistency does not impact the actual available capacity of the vdev.
+
```

Comment:
What about using a hint admonition for this content to call more attention to it, maybe the hint tip type? and Make it a snippet as you have the same content in the other article.